quickstarts/lacework/config.yml

```
@@ -13,7 +13,7 @@ description: >-
 
   If you encounter any issues, have feedback or would like more details on how to get started, please head over to support.lacework.com or send an email to adam.larson@lacework.net!
 
-logo: logo.svg
+logo: logo.jpg
```

This is the cause of the conflict. We recently changed the name of this field over to `icon` to match our internal schema. Converting it over should resolve the issue.
```suggestion
icon: logo.jpg
```